.travis.yml

```diff
@@ -34,9 +34,9 @@ install:
 script:
   - export OUTSOURCE=$PWD/build
   - export SOURCEPATH=$PWD
-  - mkdir -p $OUTSOURCE && cd $OUTSOURCE && cmake $SOURCEPATH && make && make install
-  - cd $SOURCEPATH/bin && echo "os.exit()" | lua -l color_coded
-  - cd $SOURCEPATH && rm -rf bin && mv $OUTSOURCE/clang* . && rm -rf $OUTSOURCE
-  - cmake . && make && make install
-  - cd $SOURCEPATH/bin && echo "os.exit()" | lua -l color_coded
+  - mkdir -p $OUTSOURCE && cd $OUTSOURCE && cmake $SOURCEPATH -DDOWNLOAD_CLANG=1 -DMAKE_INSTALL_RPATH_USE_LINK_PATH=ON && make && make install
+  - cd $SOURCEPATH && echo "os.exit()" | lua -l color_coded
+  - cd $SOURCEPATH && mv $OUTSOURCE/clang* . && rm -rf $OUTSOURCE
+  - cmake . -DDOWNLOAD_CLANG=1 -DMAKE_INSTALL_RPATH_USE_LINK_PATH=ON && make && make install
```

Comment:
We set `CMAKE_INSTALL_RPATH_USE_LINK_PATH` always now, so this bit can be removed, I think.